apps/src/studio/studio.js

```diff
@@ -1093,9 +1093,11 @@ Studio.init = function(config) {
   // pre-load images asynchronously
   // (to reduce the likelihood that there is a delay when images
   //  are changed at runtime)
-  preloadActorImages();
-  preloadProjectileImages();
-  preloadBackgroundImages();
+  if (config.skin.id == "studio") {
```

Comment:
Gotcha. That's fine by me, i'd just hide it behind if (skin.preloadAssets) instead of a skin.id check, and then set preloadAssets in the skin.